tasks/main.yml

```diff
@@ -16,6 +16,9 @@
       - libopenbabel-dev
       - cmake
       - python-dev
+      - python3-dev
+      - python-pip
+      - python3-pip
```

Comment:
indeed, this seems a bit "wrong"
we should be able to install six for the right python version without having to explicitly install python3

Reply:
Yeh I just don't know what the minimal install requirements are and its difficult to properly test. But, as mentioned, is at least working